starter-code/pizza.js

```diff
@@ -1 +1,63 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function () {
+
+    var $greenPepper = $('.green-pepper');
+    var $mushroom = $('.mushroom');
+    var $pep = $('.pep');
+
+
+    $('.btn-green-peppers').on('click', greenPepper);
+    $('.btn-mushrooms').on('click', mushrooms);
+    $('.btn-pepperonni').on('click', pepperonni);
+
+
+
+    function greenPepper() {
+        $('.green-pepper').toggle();
+    }
+    function mushrooms() {
+        $('.mushroom').toggle();
+    }
+
+    function pepperonni() {
+        $('.pep').toggle();
+    }
+
+    $('.btn-sauce').on('click', whiteSauce);
+    
+    
+    function whiteSauce() {
+        var sauceName = $('.btn-sauce').text();
+        
+        if (sauceName === 'White sauce') {
+            $('.sauce').removeClass('sauce-white')
```

Comment:
Interesting approach but it would be better to use toggleClass() which would avoid the need to check the current class. If it's on, it goes off, else it goes on. Later on the exercise you would have needed to control the initial state